### PR TITLE
hammerheadcaf: Update blobs

### PR DESCRIPTION
**Pull request by ghost**
* Widevine-related firmware files from bullhead N4F26I factory image (mistakenly picked from binary package in c2f3e00 commit)
 * Restored all other files to their previous state. Our attempt to get L1 provisioning failed, so we'll only update the firmware for now.

Change-Id: I20200b6d2778355aade71fc67dc161aac583a223

==================================================

Our approach to get Widevine DRM provisioning / security level L3 failed. This commit and its [counterpart in the device-tree](https://review.lineageos.org/#/c/159344/) restore all the files to their previous state except for the firmware files (.mdt and .b0*). These are from bullhead. hammerhead (AOSP) did the same and apparently all kinds of DRM-content work on L3 (when only the firmware files have been exchanged).

TL;DR:
DRM is working; the sha1 sums of the pinned files match (after I submit the commit mentioned above).

### CODE REVIEW

**Comment by ghost** [comment]
@mikeNG Could you please merge this one as well? Thanks a lot! 👍 :)

**Comment by ghost** [comment]
Thank you very much @razorloves ! :)
